app/views/portal/portal/home.html.erb

```diff
@@ -92,8 +92,12 @@
         </div>
       </div>
     <% end %>
-    <div class="spacing-above-60">
+  <% end %>
+
+  <div class="spacing-above-60">
+    <% if @tax_returns.present? %>
       <%= link_to t(".submit_documents"), portal_upload_documents_path, class: "button button--wide" %>
-    </div>
-  <%end %>
+    <% end %>
+    <%= link_to t(".send_message"), new_portal_message_path, class: "button button--wide button--primary" %>
```

Comment:
the placement of the button might change a little with the changes from this refactoring branch that was merged https://github.com/codeforamerica/vita-min/pull/1007